student-information-management-application/src/main/test/com/student/information/management/add/data/student/dao/impl/StudentDaoImplTest.java

```diff
@@ -0,0 +1,56 @@
+package com.student.information.management.add.data.student.dao.impl;
+
+import com.student.information.management.add.app.model.student.Student;
+import com.student.information.management.add.data.student.dao.StudentDao;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+/**
+ * This is the Student Dao Impl Test.
+ */
+public class StudentDaoImplTest {
+    @Mock
+    private StudentDao studentDao;
+    private List<Student> students;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
why is this necessary?

Reply:
It is necessary po for testing the addStudent. If the @Test will pass or not.

Reply:
Already Fix po.